unime/src/routes/(app)/me/+page.svelte

```diff
@@ -34,6 +39,12 @@
     }
   }
 
+  // Explicitly listen for dark mode changes, so that icons can react to it
```

Comment:
The icon's own styles should adapt their styling based soley on the presense or absense of the global `dark` class, not on a flag. The `dark` class on the html element is the flag.